### PR TITLE
Content supplement. 

### PR DESCRIPTION
**Pull request by jimcat8**
#### Author checklist (Completed by original Author)
- [x] Contribution a good fit for the Rocky project? Title and Author MetaTags inserted ?
- [ ] Is this a non-English contribution? 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by jimcat8** [comment]
Do not merge for the time being, there are still contents that need to be modified and added.

**Comment by sspencerwire** [comment]
> Do not merge for the time being, there are still contents that need to be modified and added.

Hey @jimcat8, Since you don't want this merged for right now, I've converted it to a draft. That way it won't be accidentally merged before you are ready.

**Comment by sspencerwire** [comment]
@SergeCroise and @jimcat8 , What is the status of these two (#'s 927 and 929)?  Currently, they are both in draft mode, so I'm not doing anything with them, but is there a crossover conflict between the two?

**Comment by SergeCroise** [comment]
> @SergeCroise and @jimcat8 , What is the status of these two (#'s 927 and 929)? Currently, they are both in draft mode, so I'm not doing anything with them, but is there a crossover conflict between the two?

i have reverted my modifications in the conflicting file about file systems,
i will review the file after PR# 927 is finished if you like.

**Comment by sspencerwire** [comment]
> > Do not merge for the time being, there are still contents that need to be modified and added.
> 
> Hey @jimcat8, Since you don't want this merged for right now, I've converted it to a draft. That way it won't be accidentally merged before you are ready.

Hey @jimcat8, are you ready for this one to be reviewed and merged?  It's still in draft form (I know, I put it there! *grins*) so let me know if you are ready for it to be merged.

File: docs/books/admin_guide/07-file-systems.md [hunk 1]
```diff
@@ -34,36 +34,58 @@ Partitioning will allow the installation of several operating systems because it
 
 The division of the physical disk into partitioned volumes is recorded in the partition table, stored in the first sector of the disk (MBR: _Master Boot Record_).
 
-The same physical disk can be divided into a maximum of 4 partitions:
+For **MBR** partition table types,, the same physical disk can be divided into a maximum of 4 partitions:
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:37]:
```suggestion
For **MBR** partition table types, the same physical disk can be divided into a maximum of 4 partitions:
```

File: docs/books/admin_guide/07-file-systems.md [hunk 2]
```diff
@@ -34,36 +34,58 @@ Partitioning will allow the installation of several operating systems because it
...
 
-* *Primary* (or main)
-* *Extended*
+* *Primary partition* (or main partition)
+* *Extended partition*
 
 !!! Warning
 
-    There can be only one extended partition per physical disk. In order to benefit from additional drives, the extended partition can be split into logical partitions
+    There can be only one extended partition per physical disk, That is, a physical disk can have in the MBR partition table up to:
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:44]:
```suggestion
    There can be only one extended partition per physical disk, that is, a physical disk can have in the MBR partition table up to:
```

**Review by SergeCroise — APPROVED** [review]

File: docs/books/admin_guide/07-file-systems.md [hunk 3]
```diff
@@ -73,14 +95,14 @@ sudo fdisk -l /dev/sdc2
 
 ### `parted` command
 
-The `parted` (_partition editor_) command is able to partition a disk.
+The `parted` (_partition editor_) command is able to partition a disk, it solves the shortcomings of `fdisk`, So we recommend that you use the `parted` command even more.
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:98]: shortcomings of `fdisk`. So we recommend that

Comment (sspencerwire) [reply docs/books/admin_guide/07-file-systems.md:98]: Ahh... one thing @SergeCroise : that would be a lower-case 's': "so we recommend that you use the `parted` command even more."

File: docs/books/admin_guide/07-file-systems.md [hunk 4]
```diff
@@ -504,14 +564,27 @@ proc                           /proc     proc    defaults        0   0
 | 5      | Enable or disable backup management (0:not backed up, 1:backed up)                                |
 | 6      | Check order when checking the SF with the `fsck` command (0:no check, 1:priority, 2:not priority) |
 
-The `mount -a` command allows new mounts to be taken into account without rebooting.
-They are then written to the `/etc/mtab` file which contains the current mounts.
+The `mount -a` command allows you to mount automatically based on the contents of the configuration file `/etc/fstab`, The mounted information is then written to `/etc/mtab`.
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:567]:
```suggestion
The `mount -a` command allows you to mount automatically based on the contents of the configuration file `/etc/fstab`. The mounted information is then written to `/etc/mtab`.
```

File: docs/books/admin_guide/07-file-systems.md [hunk 5]
```diff
@@ -614,55 +683,50 @@ Examples of extension agreements:
...
+Shell > mkdir /tmp/t1
+Shell > ls -ldi /tmp/t1
+1179657 drwxr-xr-x 2 root root 4096 Nov 14 18:41 /tmp/t1
 ```
 
-The letter `d` at the beginning of the rights group indicates that it is a directory type file.
-
 #### Special files
 
 In order to communicate with peripherals (hard disks, printers, ...), Linux uses interface files called special files (_device file_ or _special file_). They allow identification by the peripherals.
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:732]:
```suggestion
In order to communicate with peripherals (hard disks, printers...), Linux uses interface files called special files (_device file_ or _special file_). They allow identification by the peripherals.
```

File: docs/books/admin_guide/07-file-systems.md [hunk 6]
```diff
@@ -785,46 +815,41 @@ These controls are functions of:
...
-The user types associated with file access rights are:
+The description of **directory permissions** is as follows:
 
-* **u**ser_ (owner) ;
-* **g**roup_ (owner group);
-* **o**thers (others users);
+| Directory permissions | Description                                                                        |
+| :---------------:     | -------------                                                                      |
+| r                     | Read. Allows reading the contents of a directory (`ls -R`).                        |
+| w                     | Write. Allows you to create, and delete  files/directories in this directory, such as commands `mkdir`, `rmdir`, `rm`, `touch`, and so on.  |
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:834]:
```suggestion
| w                     | Write. Allows you to create, and delete files/directories in this directory, such as commands `mkdir`, `rmdir`, `rm`, `touch`, and so on.  |
```

File: docs/books/admin_guide/07-file-systems.md [hunk 7]
```diff
@@ -1111,13 +1001,18 @@ $ ls -la  umask_025
 
 !!! Warning
 
-    `umask` does not affect existing files.
+    `umask` does not affect existing files. `umask -S` displays the file rights (without the execute right) of the files that will be created. So it is not the display of the mask used to subtract the maximum value.
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:1004]:
```suggestion
    `umask` does not affect existing files. `umask -S` displays the file rights (without the execute right) of the files that will be created. So, it is not the display of the mask used to subtract the maximum value.
```

**Comment by sspencerwire** [comment]
@jimcat8, did you decide you just wanted me to pull this down and edit as necessary? Just let me know. I thought from your comment yesterday that you were going to do it.

**Comment by jimcat8** [comment]
> @jimcat8, did you decide you just wanted me to pull this down and edit as necessary? Just let me know. I thought from your comment yesterday that you were going to do it.

Yes, you can pull this PR and edit it.

File: docs/books/admin_guide/07-file-systems.md [hunk 8]
```diff
@@ -132,36 +154,43 @@ The preparation, without _LVM_, of the physical media goes through five steps:
...
-    LVM is only managed by the operating system. Therefore the _BIOS_ needs at least one partition without LVM to boot.
+**VG(Volume Group)**: Similar to physical disks in a standard partition, a VG consists of one or more PV.
 
-### Volume groups
+**LV(Logical Volume)**: Similar to hard disk partitions in standard partitions, LV is built on top of VG. You can set up a file system on LV.
 
-The physical volumes *PV* _Physical Volumes_ (from partitions) are combined into volume groups *VG*.
-Each *VG* represents disk space that can be partitioned into *LV* _Logical Volumes_.
-*Extension* is the smallest unit of fixed-size space that can be allocated.
+<b><font color="blue">PE</font></b>: The smallest unit of storage that can be allocated in a Physical Volume, default to <b>4MB</b>. You can specify an additional size
```
Comment (SergeCroise) [inline docs/books/admin_guide/07-file-systems.md:175]:
```suggestion
<b><font color="blue">PE</font></b>: The smallest unit of storage that can be allocated in a Physical Volume, default to <b>4MB</b>. You can specify an additional size.
```

**Pull request merged**